docs/sources/k6/next/using-k6-browser/recommended-practices/preventing-too-many-time-series-issue.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Preventing too many time series issue'
+description: 'A guide on how to prevent the `too many time series` issue when using k6 browser.'
+weight: 05
+---
+
+# Preventing too many time series issue
+
+Modern websites are complex and make a large number of requests to function as intended by their developers. These requests no longer serve only content for display to the end user but also retrieve insights, analytics, advertisements, and cache-busting purposes. Such requests are usually generated dynamically and may contain frequently changing IDs, posing challenges when correlating and analyzing your K6 test results.
+
+When load testing a website using the k6 browser module, these dynamic requests can result in a high number of similar-looking requests, making it difficult to correlate them and extract valuable insights. This can also lead to test errors, such as a _too-many-metrics_ error, due to high cardinality from metrics tagged with similar but dynamically changing URLs.
```

Comment:
```suggestion
When load testing a website using the k6 browser module, these dynamic requests can result in a high number of similar-looking requests, making it difficult to correlate them and extract valuable insights. This can also lead to test errors, such as a `too-many-metrics` error, due to high cardinality from metrics tagged with similar but dynamically changing URLs.
```

Reply:
What do you think of changing the _too-many-metrics_ to `too-many-metrics`? I'm thinking that if it's a code error, it might be better to have it as a code snippet here as well.